### DISCUSSION TITLE
How to get stream by selecting stream profile from axis camera using axis sdk?

### DISCUSSION THREAD

**Discussion by maazullah96** [AxisCommunications/discussions#221] [ACAP]
I'm working on a project that requires me to retrieve a stream from a specific stream profile. However, I couldn't find any relevant functions or modules that could help me with this task. Can you guide me on how I can retrieve a stream from a specific stream profile?

**Comment by pataxis** [comment]
Duplicate of https://github.com/AxisCommunications/acap3-examples/discussions/205
